sklearn/svm/classes.py

```diff
@@ -1025,6 +1026,10 @@ class OneClassSVM(BaseLibSVM):
         generator; If None, the random number generator is the RandomState
```

Comment:
I think this docstring should be changed to ignored. Unlike other deprecated parameters, it actually has no effect.